spec/workers/notifications/new_follower_worker_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe Notifications::NewFollowerWorker, type: :worker do
+  let(:follow_data) { { followable_type: "User", followable_id: 1, follower_id: 2 } }
+  let(:worker) { subject }
+
+  include_examples "#enqueues_on_correct_queue", "medium_priority", [{}, {}]
+
+  describe "#perform" do
+    let(:new_follower_service) { Notifications::NewFollower::Send }
+
+    before { allow(new_follower_service).to receive(:call) }
+
+    it "calls the service" do
+      worker.perform(follow_data)
+      allow(new_follower_service).to receive(:call).with(follow_data).once
```

Comment:
I'd keep the `.with(follow_data, false).once` part that was present in the previous job spec, just to be sure that if `is_read`'s default gets switch to true the test will fail :)